src/modules/Brightness/Module.js

```diff
@@ -4,6 +4,8 @@
 
 module.exports = function Brightness(options,UI){
 
+    options.brightness = parseInt(options.brightness) || 100
```

Comment:
Okay, wouldn't this give an error if options.brightness is undefined, which kind of defeats the purpose of having this check in the first place?

Reply:
I tried it out, parseInt(undefined) gives a NaN so options.brightness takes value 100, it doesn't give an error but it should have been inside the draw function I will correct that. Thanks!